CMSgov/design-system
# [MCT-6718] Adding medicare specific icons

## Summary
Resolves #1683
Adds 4 new medicare specific icons

### Added
|name |screenshot  |
--- | --- |
|`<CheckShieldIcon />` |![image](https://user-images.githubusercontent.com/24252610/161794615-f5e66641-85bf-445b-9bb8-87d2cb07be3c.png)|
|`<DrugsIcon />`|![image](https://user-images.githubusercontent.com/24252610/161794770-014610b9-6b12-4aa7-b80e-f8b21a64796f.png)|
|`<PiggyBankIcon />`|![image](https://user-images.githubusercontent.com/24252610/161794670-f001c7b6-d190-4f1f-ab30-b27b4b95c1e8.png)|
|`<PharmacyIcon />`|![image](https://user-images.githubusercontent.com/24252610/161794714-437297e9-c505-45bf-b80a-e0d19fd31050.png)|

## How to test
- Boot up the Medicare Design System docsite
- Navigate to `/components/icon/` and see the new icons added at the top of the page
